### PR TITLE
Get blocks

### PR DESCRIPTION
**Pull request by aakilfernandes**
Adds a .getBlocks function

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -297,6 +298,48 @@ Blockchain.prototype.getBlock = function (hash, cb) {
 }
 
 /**
+ *Gets a block by its hash
+ * @method getBlocks
+ * @param blockId - the block's hash or number
+ * @param {Function} cb - the callback function
+ */
+Blockchain.prototype.getBlocks = function (blockId, maxBlocks, skip, reverse, cb) {
```
Comment (wanderer) [inline index.js]: where would skipping be useful?

File: index.js [hunk 2]
```diff
@@ -297,6 +298,48 @@ Blockchain.prototype.getBlock = function (hash, cb) {
 }
 
 /**
+ *Gets a block by its hash
```
Comment (wanderer) [inline index.js]: can you update the comments?

File: index.js [hunk 3]
```diff
@@ -297,6 +298,48 @@ Blockchain.prototype.getBlock = function (hash, cb) {
...
+
+      var nextBlockNumber = ethUtil.bufferToInt(block.header.number) + (reverse ? -1 : 1)
+      
+      if(i !== 0 && skip && i % (skip + 1) !== 0)
+        return nextBlock(nextBlockNumber)
+    
+      blocks.push(block)
+
+      if(blocks.length === maxBlocks)
+        return cb(null, blocks)
```
Comment (wanderer) [inline index.js]: I don't think this is allowed by the lint. But I might be wrong. can you run `npm run lint`?

File: index.js (hunk 1 above)
Comment (aakilfernandes) [inline index.js]: Not sure why but eth62's GetBlockHeaders has a skip option https://github.com/ethereum/wiki/wiki/Ethereum-Wire-Protocol#new-model-syncing-pv62

Comment (wanderer) [inline index.js]: ah ok fair enough

**Comment by wanderer** [comment]
hey @aakilfernandes I don't have time to properly review this. Can you merge it after you get the tests passing?

**Comment by kumavis** [comment]
I'll resolve this

**Pull request merged**